app/controllers/staff/organisations_controller.rb

```diff
@@ -11,7 +11,7 @@ def show
     authorize organisation
 
     @organisation_presenter = OrganisationPresenter.new(organisation)
-    activities = policy_scope(Activity.funds).includes(:organisation).where(organisation: organisation)
+    activities = policy_scope(Activity.fund).includes(:organisation).where(organisation: organisation)
```

Comment:
I'm surprised this works without being plural!

Reply:
Plural didn't work at all